.editorconfig

```diff
@@ -7,12 +7,120 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+max_line_length = 150
```

Comment:
Most of these things are simply the result of exporting the code style settings from IJ as they were already used in the project (essentially, the Kotlin official style)